test/testSetup.ts

```diff
@@ -74,82 +97,86 @@ export class TestSetup {
     if (!fs.existsSync(this.tempFolderPath)) {
-      await utilities.createFolder(this.tempFolderPath);
-      await utilities.pause(1);
+      utilities.createFolder(this.tempFolderPath);
     }
 
     utilities.log(`${this.testSuiteSuffixName} - ...finished setUpTestingEnvironment()`);
     utilities.log('');
   }
 
-  public async createProject(scratchOrgEdition: string): Promise<void> {
+  public async createInitialProject(scratchOrgEdition: string): Promise<void> {
     utilities.log('');
-    utilities.log(`${this.testSuiteSuffixName} - Starting createProject()...`);
-
-    const workbench = await (await browser.getWorkbench()).wait();
+    utilities.log(`${this.testSuiteSuffixName} - Starting createInitialProject()...`);
 
     // If you are not in a VSCode project, the Salesforce extensions are not running
     // Force the CLI integration extension to load before creating the project
-    await utilities.runCommandFromCommandPrompt(workbench, 'Developer: Show Running Extensions', 5);
-    await utilities.runCommandFromCommandPrompt(workbench, 'SFDX: Create Project', 5);
+    const workbench = await utilities.getWorkbench();
+    await utilities.showRunningExtensions();
+    const prompt = await workbench.executeQuickPick('SFDX: Create Project');
+    await utilities.waitForQuickPick(prompt, 'Standard', {
+      msg: 'Expected extension salesforcedx-core to be available 5 seconds',
+      timeout: 5_000
+    });
     await browser.keys(['Escape']);
     await utilities.pause(1);
     await browser.keys(['Escape']);
 
-    // Do not continue until we verify CLI Integration extension is present and running
-    let coreExtensionWasFound = false;
-    do {
-      await utilities.pause(10);
+    const coreIsActive = await utilities.verifyExtensionsAreRunning(
+      utilities
+        .getExtensionsToVerifyActive()
+        .filter((ext) => ext.extensionId === 'salesforcedx-vscode-core')
+    );
+
+    if (!coreIsActive) {
+      fail('Expected core extension to be active after 20 seconds');
+    }
 
-      coreExtensionWasFound = await utilities.findExtensionInRunningExtensionsList(
-        workbench,
-        'salesforcedx-vscode-core'
-      );
-    } while (coreExtensionWasFound === false);
     utilities.log(`${this.testSuiteSuffixName} - Ready to create the standard project`);
 
-    this.prompt = await utilities.runCommandFromCommandPrompt(workbench, 'SFDX: Create Project', 5);
-    // Selecting "SFDX: Create Project" causes the extension to be loaded, and this takes a while.
+    await this.createProject(workbench, this.tempProjectName, scratchOrgEdition);
+
+    // Extra config needed for Apex LSP on GHA
+    const os = process.platform;
+    if (os === 'darwin') {
+      this.setJavaHomeConfigEntry();
+    }
+
+    utilities.log(`${this.testSuiteSuffixName} - ...finished createInitialProject()`);
+    utilities.log('');
+  }
 
+  public async createProject(workbench: Workbench, projectName: string, scratchOrgEdition: string) {
```

Comment:
`workbench` is unused in this function.

```suggestion
  public async createProject(projectName: string, scratchOrgEdition: string) {
```